overlay.d/05core/usr/lib/dracut/modules.d/35coreos-ignition/coreos-unique-boot.service

```
@@ -1,5 +1,5 @@
 [Unit]
-Description=Ensure filesystem labeled `boot` is unique
+Description=Ensure Filesystem Labeled `boot` Is Unique
```

Minor: while we're here, should we rename this

```suggestion
Description=Ensure Unique `boot` Filesystem Label
```

to make it less wordy?